### PR TITLE
created form to let users ask for partnership

### PR DESCRIPTION
**Pull request by LucyMerlier**
quick demo : https://www.loom.com/share/ab6956d5eb304f5b95831499e91db8a9

### CODE REVIEW

**Review by YannRaulet — DISMISSED** [review]

File: src/DataClass/Partnership.php [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+     * @Assert\Positive()
+     */
+    private int $workforce;
+
+    /**
+     * @Assert\Length(max="2000")
+     */
+    private ?string $message = null;
+
+    private string $submitDate;
```
Comment (RomainClair) [inline src/DataClass/Partnership.php]: You probably shoud use the build-in doctrine/php date type

File: src/DataClass/Partnership.php [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+    /**
+     * @Assert\NotBlank()
+     */
+    private string $telephone;
+
+    /**
+     * @Assert\NotBlank()
+     */
+    private string $email;
```
Comment (RomainClair) [inline src/DataClass/Partnership.php:58]: There is an email validation constraint in symfony

File: src/DataClass/Partnership.php [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Length(min="2", max="255")
+     */
+    private string $businessName;
+
+    /**
+     * @Assert\NotBlank()
+     */
+    private string $expertise;
```
Comment (RomainClair) [inline src/DataClass/Partnership.php:23]: Your expertise seems to be one value in a list of given choice. You could validate that it is one of this choices.

**Review by YannRaulet — APPROVED** [review]

File: src/DataClass/Partnership.php (hunk 3 above)
Comment (RomainClair) [reply src/DataClass/Partnership.php:23]: https://symfony.com/doc/current/reference/constraints/Choice.html would be better than regex in that case

**Review by RomainClair — APPROVED** [review]

**Pull request merged**